js/schedule.js

```diff
@@ -347,13 +370,28 @@ function Schedule(CUSTOM_CONFIG) {
     $('#session-detail-wrapper').remove();
   }
 
+  schedule.saveScrollState = function(page, scrollY) {
```

Comment:
Ah, I was just thinking that it's supported in even older browsers, thanks for pointing that out, updated!